public/docs/ts/latest/guide/server-communication.jade

```diff
@@ -45,10 +45,10 @@ h1 Demos
 p  This chapter describes server communication with the help of the following demos
 ul
   block demos-list
-    li HTTP client: Tour of Heroes with Observables
-    li HTTP client: Tour of Heroes with #{_Promise}s
-    li JSONP client: Wikipedia to fetch data from a service that doesn't support CORS
-    li JSONP client: Wikipedia using observable operators to reduce server calls
+    li #[a(href="#http-client") HTTP client: Tour of Heroes with Observables]
```

Comment:
As currently written, this list might be better off without the links, since the targets are section heads with different levels that only sometimes match the names here.